src/utilities/core/Logger.hpp

```diff
@@ -94,40 +93,10 @@ class UTILITIES_API LoggerImpl
   void addTimeStampToLogger();
 
  private:
-  /// private constructor
-  LoggerImpl();
-
-  mutable std::shared_mutex m_mutex;
-
-  /// standard out logger
-  LogSink m_standardOutLogger;
-
-  /// standard err logger
-  LogSink m_standardErrLogger;
+  Logger();
+  ~Logger() = default;
 
-  /// map of std::string to logger
-  using LoggerMapType = std::map<std::string, LoggerType, openstudio::IstringCompare>;
-  LoggerMapType m_loggerMap;
-
-  /// current sinks, kept here so don't destruct when LogSink wrapper goes out of scope
-  using SinkSetType = std::set<boost::shared_ptr<LogSinkBackend>>;
-  SinkSetType m_sinks;
-};
-
-class UTILITIES_API Logger
-{
- public:
-  Logger() = delete;
-
-  static LoggerImpl& instance() {
-    if (!obj) {
-      obj = std::shared_ptr<LoggerImpl>(new LoggerImpl());
-    }
-    return *obj;
-  }
-
- private:
-  static std::shared_ptr<LoggerImpl> obj;
+  std::shared_ptr<detail::Logger_Impl> m_impl;
```

Comment:
Given that the Logger is a Meyers singleton, I'm not sure whether the Logger_Impl class is really needed at all. But it feels familiar at least, and can allow us to shorten the public includes

Reply:
I agree, I don't think we really need an Impl at all.